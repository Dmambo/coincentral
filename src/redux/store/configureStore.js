import { configureStore } from '@reduxjs/toolkit';
import cryptoReducer from '../reducer/crypto';

const store = configureStore({
  reducer: {
    crypto: cryptoReducer,
  },
});

export default store;
