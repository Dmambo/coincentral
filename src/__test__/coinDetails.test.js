import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import CoinDetails from '../components/coinDetails/coinDetails';
import store from '../redux/store/configureStore';

test('renders coinDetails component', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <CoinDetails />
    </Provider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
