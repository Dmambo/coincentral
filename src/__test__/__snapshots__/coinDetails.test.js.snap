// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coinDetails component 1`] = `
<DocumentFragment>
  <div>
    No coin selected
  </div>
</DocumentFragment>
`;
